### PR TITLE
A9: Float Layout

### PR DESCRIPTION
**Pull request by Angelbt12**
activity completed successfully

### CODE REVIEW

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  <body>
+      <div class="sidebar">
+        <div class="column">
+          <div class="avatar"></div>
+        </div>
+        <h3>Angel Baracaldo</h3>
+        <div>
+          <div class="menu">
+            <br />
+            <a href="#" class="enlace">
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:17]: Note that class enlace it is not necessary, plesase delete that class and add propirtes to "a" tag

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    <link rel="stylesheet" href="A9.css" />
+  </head>
+  <body>
+    <div class="sidebar">
+      <div class="column">
+        <div class="avatar"></div>
+      </div>
+      <h3>Angel Baracaldo</h3>
+      <div>
+        <div class="menu">
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:15]: 1. avoid use class for one component.
2. use a list to contain all a tags in this nav
3. with a list you don't need a class to stylize the container
Please refactor it

Comment (Angelbt12) [reply src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:15]: ok I understand, I already made the changes

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html (hunk 1 above)
Comment (Angelbt12) [reply src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html]: ok I understand, I already made the changes

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+          <hr />
+          <p>My playlist #1</p>
+          <p>Christmas Tracks</p>
+          <p>Other Remix</p>
+      </ul>
+      </div>
+    </div>
+    <div class="content">
+      <img src="img/img.PNG" alt="" />
+      <div class="text-container">
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:72]: Avoid use class for just one component like classes "content" "text-container".
Please refactor using pseudoselectors

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Floats</title>
+    <link rel="stylesheet" href="A9.css" />
+  </head>
+  <body>
+    <div class="sidebar">
+        <div class="avatar"></div>
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:10]: Avoid creating or using a class for a single component like,
sidebar, avatar, content, text-container.
Please refactor using pseudoclasses or pseudoelements

**Review by luisalejandrosalazar — COMMENTED** [review]
Avoid creating or using a class for a single component like,
sidebar, avatar, content, text-container.
Please refactor using pseudoclasses or pseudoelements

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+          <a >
+            <i class="fa-solid fa-bars"></i>
+            <span>Records</span>
+          </a>
+
+          <a >
+            <i class="fa-solid fa-square-plus"></i>
+            <span>Create Playlist</span>
+          </a>
+          <hr />
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:63]: Remember that it is bad practice to use br to add space between components, or use hr to draw a line. You should use css properties instead.

File: src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+    <title>Floats</title>
+    <link rel="stylesheet" href="A9.css" />
+  </head>
+  <body>
+    <div >
+        <div ></div>
+      <h3>Angel Baracaldo</h3>
+      <div>
+        <ul>
+          <br />
```
Comment (luisalejandrosalazar) [inline src/ab/html/A2-Markup-a-readme/A9-Float Layout/A9.html:14]: Remember that it is bad practice to use br to add space between components, or use hr to draw a line. You should use css properties instead.

**Review by luisalejandrosalazar — APPROVED** [review]
Approved

**Pull request merged**